_posts/2019-11-01-snowplow-r117-biskupin/index.md

```diff
@@ -0,0 +1,247 @@
+5. [Support to the spot market for core instances](#support-to-the-spot-market-for-core-instances)
+
+<!--more-->
+
+## TLS port binding and certificate
+
+### TLS/SSL Certificate termination
+As an additional security measure it is now possible to terminate TLS/SSL connection directly within `scala-stream-collector` using a battle-tested [Lightbend SSL Config](https://lightbend.github.io/ssl-config/index.html).
+We have introduced several new configuration parameters in order to accommodate different workflows and configurations.
+There are two configuration sections that can be overriden in order to achieve the expected workflow: `collector.ssl` and `ssl-config`.
+
+The former is a high-level section that allows:
+- `collector.ssl.enable` - turn on ssl termination
+- `collector.ssl.redirect` - whether automatic upgrade from http to https should be performed
+- `collector.ssl.port` - port on which TLS/SSL server should be started
+
+The latter allows for low-level TLS/SSL configuration exposed by [Lightbend SSL Config](https://lightbend.github.io/ssl-config/index.html).
+​
+For example to start up an ssl-enabled, auto-upgrade server, following config can be used:
+```json
+ssl {
+  enable = true
+  redirect = true
+  port = 443
+}
+```
+However, this configuration will use environment-defined JVM-attached certificates. In order to override the default behaviour and use a custom certificate, the low-level section can be defined as:
+```json
+ssl-config {
+  keyManager = {
+    stores = [
+      {type = "PKCS12", classpath = false, path = ${CERT_FILE}, password = "pass" }
+    ]
+  }
+}
+```
+​
+### Default redirect endpoint disabling
+Another simple, but important security improvement is a possibility to disable the default event submission redirect. This allows users to disable the default endpoint in favour of custom user-defined url. 
+For example, following configuration will only allow redirects for custom-defined `/com.acme/redirect-me` endpoint, whereas the default `/r/tp2` will not be available.
+```json
+enableDefaultRedirect = false
+paths {
+  "/com.acme/redirect-me" = "/r/tp2"
+}
+```
+Note, that for backwards-compatibility, the endpoint is still exposed by default.
+
+## Referer parser refreshment
+
+Referer parser is a library that contains a list of possible referer hosts. We keep maintenance of `0.3.x` branch with backporting new identified hosts into old `0.3.0` version. Several new hosts were identified and added to the [`referer.yml`](https://s3-eu-west-1.amazonaws.com/snowplow-hosted-assets/third-party/referer-parser/referers-latest.yml)
+
+## IPv6 anonymization
+
+This release adds support for anonymization of IPv6 addresses. `anonSegments` parameter is added to enrichment configuration. It describes how many segments should be anonymized.
+
+The schema for the configuration of the enrichment has been updated to version `1-0-1`:
+
+```json
```

Comment:
Why we post the whole config file here, but only mention a SchemaVer for fingerpritn enrichment? I'd post full configs only in Upgrading section.